site/gatsby-site/blog/using-ai-to-connect-ai-incidents/index.es.mdx

```diff
@@ -10,95 +10,96 @@ slug: '/blog/using-ai-to-connect-ai-incidents'
 
-To support scalable, asynchronous, and fast similarity correlation, this implementation uses a full stack of AWS tools, all built and deployed using AWS CDK v2. The AWS side of the solution primarily consists of a collection of AWS Lambdas and an AWS API Gateway RESTful HTTP API. This system and build-chain was designed to be highly modular and expandable, so that new Lambda functions or API features could be added at any time. In addition, the Longformer model can be easily swapped out for another pre-trained or fine-tuned model, so that future development can go in whatever direction best supports the growth of the database.
+Para admitir una correlación de similitud escalable, asíncrona y rápida, esta implementación utiliza una pila completa de herramientas de AWS, todas creadas e implementadas con AWS CDK v2. El lado de AWS de la solución consiste principalmente en una colección de AWS Lambdas y una API RESTful HTTP de AWS API Gateway. Este sistema y la cadena de compilación se diseñaron para ser altamente modulares y ampliables, de modo que se pudieran agregar nuevas funciones de Lambda o funciones de API en cualquier momento. Además, el modelo Longformer se puede cambiar fácilmente por otro modelo previamente entrenado o ajustado, de modo que el desarrollo futuro pueda ir en la dirección que mejor apoye el crecimiento de la base de datos.
 
-![An architecture diagram showing the following connections: 1. AWS Lambda → 2. Pretrained LongFormer Model (.bin), 1. AWS Lambda → 3. AIID DB State (CLS Means), 4. Amazon ECR → 1. AWS Lambda, 1. AWS Lambda ↔ 5. Amazon EFS, 6. Amazon API Gateway (Public) ↔ 1. AWS Lambda, 6. Amazon API Gateway (Public) → Users, 6. Amazon API Gateway (Public) → AIID Applications. Items 1–3 are inside a box labelled “Lambda Docker Container.” Items 1–5 are inside a box labelled VPC 10.0.0/16. On the border of that box is an icon showing a doorframe labelled “Internet Gateway.” Within that box is another marked by a lock icon labelled “Public Subnet.” Items 1–6 are inside a box labeled “AWS cloud.”](/images/nlp-lambdas-aws-architecture.png)
+![Un diagrama de arquitectura que muestra las siguientes conexiones: 1. AWS Lambda → 2. Modelo LongFormer preentrenado (.bin), 1. AWS Lambda → 3. Estado de base de datos AIID (CLS Means), 4. Amazon ECR → 1. AWS Lambda, 1. AWS Lambda ↔ 5. Amazon EFS, 6. Amazon API Gateway (público) ↔ 1. AWS Lambda, 6. Amazon API Gateway (público) → Usuarios, 6. Amazon API Gateway (público) → Aplicaciones AIID. Los elementos 1 a 3 están dentro de un cuadro con la etiqueta "Contenedor Lambda Docker". Los elementos del 1 al 5 están dentro de un cuadro con la etiqueta VPC 10.0.0/16. En el borde de ese cuadro hay un ícono que muestra un marco de puerta con la etiqueta "Internet Gateway". Dentro de ese cuadro hay otro marcado con un icono de candado con la etiqueta "Subred pública". Los elementos del 1 al 6 están dentro de un cuadro etiquetado como "nube de AWS".](/images/nlp-lambdas-aws-architecture.png)
```

Comment:
Sorry, not sure I follow that

Reply:
Oh, ok. I just meant that the translated labels in the alt text, like "2. Modelo LongFormer preentrenado (.bin)" might be useful to readers who aren't using screen readers, since the diagram is in English. So we could make the labels visible by wrapping the image in a `<figure>` and adding the labels in a `<figcaption>`, something like this:

```jsx
<figure>
  ![An architecture diagram...](/images/nlp-lambdas-aws-architecture.png)

  <figcaption>
     (2) Modelo LongFormer preentrenado (3) Estado de base de datos AIID (CLS Means)...
  </figcaption>
</figure>
```